src/uv.cc

```diff
@@ -60,15 +60,6 @@ using v8::Value;
 
 void ErrName(const FunctionCallbackInfo<Value>& args) {
```

Comment:
I think with this you can simply delete this C++ implementation now?

Reply:
Yeap, deleted on 01eb79f